README.md

```diff
@@ -277,12 +273,12 @@ git clone https://github.com/lifting-bits/remill.git --depth=1
 git clone https://github.com/lifting-bits/mcsema.git --depth=1 remill/tools/mcsema
 ```
 
-Note that for production usage you should always use a specific remill commit (`remill/tools/mcsema/.remill_commit_id`) when building mcsema. At the time of writing, it is however best to use HEAD (or at least make sure that commit `e7795be` is present in the remill branch).
+Note that for production usage you should always use a specific remill commit when building mcsema. At the time of writing, it is however best to use HEAD (or at least make sure that commit `e7795be` is present in the remill branch).
```

Comment:
Can we keep remill commit id around for manual builds?

Reply:
Will add this back to the README. Also looks like `.remill_commit_id` file was merged back in with updating commits from `master`